### PR TITLE
Display board documentation

### PR DESCRIPTION
**Pull request by manthanand**
Honestly, when this is made, there are lots of weird spaces between lines in bullet lists and all that. If there is a fix then I can change it. If it looks fine then it doesn't matter.

### CODE REVIEW

File: Docs/source/Hardware/Boards.rst [hunk 1]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+^^^^^^^^^
+
+**Main**
+++++++++
+
+*What does this circuit do?*
+    This circuit makes connections from the microcontroller to the E-Ink Display and logic 
+    analyzer pins.  
+*Why do we need it?*
+    We need this circuit in order for debugging (:term: `CAN <CAN Bus>` messages on the E-Ink
```
Comment (sugamarora99) [inline Docs/source/Hardware/Boards.rst:72]: The intended effect for this code didn't work it seems

File: Docs/source/Hardware/Boards.rst [hunk 2]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+        * Justification for selection of specific part: Standard component  
+        * `Datasheet link <https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiqg4WM6NXuAhXULc0KHcHbAzIQFjABegQIAhAC&url=https%3A%2F%2Fwww.st.com%2Fresource%2Fen%2Fdatasheet%2Fstm32f413rg.pdf&usg=AOvVaw3U9MV6EuhEcmJpsEYCrU52>`__
+        * Associated passives/components:
+            * Bypass capacitors 
+            * 8 MHz External Crystal Oscillator: used to set the Phase Lock Loop (PLL) and the system core frequency (CPU processor speed) 
+                * Using an external clock ensures for precision, which is important in a safety-critical system like the BPS.  
+            * Reset button
+            * :term:`SWD <SWD>` Programming Pins
+    * Logic Analyzer Pins (2.54 mm headers) 
+        * Description: The logic analyzer pins serve to easily test the board to check proper
```
Comment (sugamarora99) [inline Docs/source/Hardware/Boards.rst:86]: You could bold Description, Why is it necessary, etc

File: Docs/source/Hardware/Boards.rst [hunk 3]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+          what occurs to these signal lines through these pins  
+        * Justification for selection of specific part: Standard component 
+        * Associated passives/components:  
+            * SPIO Analyzer Pins 1x5: PA5, PA6, PA7, PA8, GND 
+            * GEN Analyzer Pins 1x4: PA9, PA10, PA11, GND 
+            * USART Analyzer Pins 1x3: PC6, PC7, GND 
+            * CAN Analyzer Pins 1x3: PB12, PB13, GND 
+    * E-Ink Display: 
+        * Description: Display screen. 
+        * Why is it necessary: CAN messages will be displayed on this screen.
```
Comment (sugamarora99) [inline Docs/source/Hardware/Boards.rst:98]: Same here

File: Docs/source/Hardware/Boards.rst [hunk 4]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+            * USART Analyzer Pins 1x3: PC6, PC7, GND 
+            * CAN Analyzer Pins 1x3: PB12, PB13, GND 
+    * E-Ink Display: 
+        * Description: Display screen. 
+        * Why is it necessary: CAN messages will be displayed on this screen.  
+        * Justification for selection of specific part: This display doesn't consume much power. 
+        * Associated passives/components: 
+            * 2x20 Connector 
+    * 2x20 Connector (2.54mm) 
+        * Description: Connects the STM to the E-Ink display.
```
Comment (sugamarora99) [inline Docs/source/Hardware/Boards.rst:103]: Same here

File: Docs/source/Hardware/Boards.rst [hunk 5]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+    * 2x20 Connector (2.54mm) 
+        * Description: Connects the STM to the E-Ink display. 
+        * Why is it necessary: Allows the E-Ink display to show CAN messages. 
+        * Justification for selection of specific part: Standard component 
+        * Associated passives/components: 
+            * Buttons: PB0, PB1, PB2, PB3 
+            * SPIO: PA5, PA6, PA7, PA8 
+            * GEN: PA9, PA10, PA11 
+    * Reset button 
+        * Description: Resets the STM microcontroller.
```
Comment (sugamarora99) [inline Docs/source/Hardware/Boards.rst:111]: Same here

File: Docs/source/Hardware/Boards.rst [hunk 6]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+=============
+
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-DisplayPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?State=EDIT&ProjectGUID=11b675c5-b15b-46bd-a790-bb450819a4d3>`__ 
+
+Brief Description/Purpose:
+    The purpose of this board is to display messages from the CAN interface on the E-Ink display
```
Comment (ClarkPoon) [inline Docs/source/Hardware/Boards.rst:21]: Should probably link to CAN term

File: Docs/source/Hardware/Boards.rst [hunk 7]
```diff
@@ -8,6 +8,235 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+Context
+^^^^^^^
+**Location of the Board:** On a movable arm mount attached to the south end of the board. Mounted near the driver.  
+
+List of I/O and Connections: 
+    [List of all the inputs and outputs from this board and where it is connected to. 
+    List all power signals first]
+    
+    * [Signal/Power] 
+        * [Input/Output/Bidirectional] [from/to/with] [Board/Hardware/etc.]
```
Comment (ClarkPoon) [inline Docs/source/Hardware/Boards.rst:52]: I think these are from Hannah's template and can be removed

**Review by ClarkPoon — CHANGES_REQUESTED** [review]

File: Docs/source/Hardware/Boards.rst [hunk 8]
```diff
@@ -8,6 +8,231 @@ Each board measures up to 12 battery modules and 16 temperature sensors, but is
...
+           | component such as a cell phone or laptop and 
+           | the Supplemental Battery was intended for 
+           | this purpose.
+========== ============================================== ===============================================
+
+Context
+^^^^^^^
+**Location of the Board:** On a movable arm mount attached to the south end of the board. Mounted near the driver.  
+
+List of I/O and Connections:
```
Comment (ClarkPoon) [inline Docs/source/Hardware/Boards.rst:47]: Some of the titles for bulleted lists are bold and italicized, some are just italicized, and this one is neither. Can we make them consistent?

**Review by ClarkPoon — CHANGES_REQUESTED** [review]

File: Docs/source/Hardware/Boards.rst (hunk 8 above)
Comment (manthanand) [reply Docs/source/Hardware/Boards.rst:47]: Yea, now that I think about it it's kinda weird. Sometimes if I do *word* it makes it bold but other times I have to do **word**. \_(--)_/

Comment (manthanand) [reply Docs/source/Hardware/Boards.rst:47]: No way, you can **bold** th**i**ngs in github comments.

Comment (manthanand) [reply Docs/source/Hardware/Boards.rst:47]: AAND *ITALICIZE* WHHAAAATTT

Comment (ClarkPoon) [reply Docs/source/Hardware/Boards.rst:47]: Yeah, you can do a lot in github comments. [You can even add images](https://github.com/lhr-solar/BPS/issues/207#issuecomment-699601773)

**Comment by ClarkPoon** [comment]
In your commit "Everything should be standardized by now" it says that you just changed some whitespace, and it just build the same on my machine (but with more warnings)

**Review by ClarkPoon — APPROVED** [review]

**Pull request merged**